dev-lang/idris/idris-1.3.3-r2.ebuild

```diff
@@ -57,13 +57,17 @@ RDEPEND=">=dev-haskell/aeson-0.6:=[profile?] <dev-haskell/aeson-1.5:=[profile?]
 "
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-2.4
-	test? ( dev-haskell/tagged
+	test? ( dev-haskell/cabal
```

Comment:
`test? ( dev-haskell/cabal` looks redundant to `>=dev-haskell/cabal-2.4` two lines above. Did you plan to write something else here?

Reply:
This is likely an artifact of me manually adding the test dependencies, I'll rerun hackport and make sure the deps match.